src/test/java/org/sonar/plugins/github/PullRequestIssuePostJobTest.java

```diff
@@ -167,6 +173,99 @@ public void testPullRequestAnalysisWithNewIssues() {
   }
 
   @Test
+  public void testSortIssues() {
+    ArgumentCaptor<String> commentCaptor = forClass(String.class);
+
```

Comment:
Sure, I'll check that when I'll get to the office.

By the way why mock issues and not use something like a DefaultIssue instead?